daemon/mgr/container.go

```diff
@@ -760,9 +760,10 @@ func (mgr *ContainerManager) createContainerdContainer(ctx context.Context, c *C
 		IO:             mgr.IOs.Get(c.ID),
 		RootFSProvided: c.RootFSProvided,
 		BaseFS:         c.BaseFS,
-		SnapshotID:     c.SnapshotID,
```

Comment:
I check that snapshot id is first given to `SnapshotID` , ` snapID := id`,  `SnapshotID:    snapID`, so why this field will be empty ?

Reply:
> I check that snapshot id is first given to SnapshotID , snapID := id, SnapshotID: snapID, so why this field will be empty
I also cannot figure it out why this happened, maybe some extreme cases

>  if this happened, do c.SnapshotKey() will also be empty
No, `SnapshotKey()` will return c.ID if the `SnapshotID` is empty